jfr-streaming/src/test/java/io/opentelemetry/contrib/jfr/streaming/JfrThreadCountTest.java

```diff
@@ -22,26 +22,13 @@ class JfrThreadCountTest {
       new JfrExtension(
           builder -> builder.disableAllFeatures().enableFeature(JfrFeature.THREAD_METRICS));
 
-  private static final int SAMPLING_INTERVAL = 1000;
-
-  private static void doWork() throws InterruptedException {
-    Thread.sleep(2 * SAMPLING_INTERVAL);
-  }
-
-  private static boolean isDaemon(LongPointData p) {
-    Boolean daemon = p.getAttributes().get(AttributeKey.booleanKey(DAEMON));
-    assertThat(daemon).isNotNull();
-    return daemon;
-  }
-
   @Test
   void shouldHaveJfrThreadCountEvents() throws Exception {
     // This should generate some events
     Runnable work =
         () -> {
-          // create contention between threads for one lock
           try {
-            doWork();
+            Thread.sleep(2000);
```

Comment:
I like removing `doWork()` but I thought the `2 * SAMPLING_INTERVAL` was slightly helpful.

Reply:
+1

Also - should we consider moving to Awaitility rather than explicit Thread.sleep() calls?

Reply:
> I like removing doWork() but I thought the 2 * SAMPLING_INTERVAL was slightly helpful.

Added an explanatory comment instead, which is even more useful since the reader doesn't have to guess what SAMPLING_INTERVAL represents.

> Also - should we consider moving to Awaitility rather than explicit Thread.sleep() calls?

We need to let the threads run long enough to produce `jdk.JavaThreadStatistics` JFR events to be consumed by the `ThreadCountHandler`, which awaitility doesn't help us with.